saga/README.md

```diff
@@ -0,0 +1,121 @@
+# Saga Pattern
+
+This example demonstrates how to implement the [Saga pattern](https://microservices.io/patterns/data/saga.html) for realizing distributed transactions across multiple microservices, in a safe and reliable way using change data capture.
+
+Similarly to the outbox pattern, this implementation avoids unsafe dual writes to a service's database and Apache Kafka by channeling all outgoing messages through the originating service's database and capturing them from there using CDC and Debezium.
+
+There are three services involved:
+
+* _order-service:_ originator and orchestrator of the Saga
+* _customer-service:_ approves or rejects the customer's credit needed to fulfill an incoming order
```

Comment:
is this about the customer's credit *card* I guess?

Reply:
No, it's about the limit you want to grant the customer when ordering. E.g. you might not want to accept an order over 1K for a new customer. I'll replace it with "credit limit", it's a common term.